### PR TITLE
[CMS-584] OSS Support Levels

### PR DESCRIPTION
**Pull request by kporras07**
This PR is part of the work we are doing to clearly identify support levels in our Open source Projects.

## Summary

**[Pantheon Open Source Software Support Levels](https://pantheon.io/docs/oss-support-levels)** - Create OSS Support Levels page

### Dependencies and Timing

This PR is pre-requisite for some other work we're doing in the CMS Ecosystem team but it has no pre-requisites to merge it other than getting the docs team approval.

**Release**:
- [x] Engineering review
- [X] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7130](https://pr-7130--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `61` | `57.99999999999999` | `-3` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7130` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34137/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34137/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7130--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/oss-support-levels.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Early Access projects are similar to Pilot projects, but are a little more mature, and have been opened up for use by a larger number of users. An invitation is still necessary to join an Early Access program.
+
+## Limited Availability
+
+A project in this support level is a product or feature set that is complete and can be used in all environments including production. Clients can use the new functionality without requiring an invite but it is an opt-in process. You are highly encouraged to report any bug you may find so that we can make a better product for everyone. Support in this stage is available through regular Pantheon channels.
+
+## Actively Maintained
+
+A project in this support level has been publicly launched and has several eyes over it ensuring that everything works as expected. Anyone could use this project and create issues and Pull Requests in the repository as needed and we will respond to them in a timely manner. Support in this stage is provided by the Customer Success Engineering team.
```
Comment (kporras07) [inline source/content/oss-support-levels.md:24]: @greg-1-anderson I'm guessing the same update should be made here, right? "Support in this stage is available through regular Pantheon channels."

File: source/content/oss-support-levels.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+At Pantheon we love Open Source, we have created lots of public repositories in our [Github account](https://github.com/orgs/pantheon-systems/repositories) and we have started tagging them with support level badges. This page describes each of those levels and what should you expect when using those repositories.
+
+## Pilot
+
+A project in Pilot support level is a new product or feature set that we are working on. The code is publicly available but if there are any accompaning platform components you will need an invite through the project pilot program. Even though we make our best effort, there may be bugs and we need your help to identify them so that we can fix them. The product or feature set may not be feature complete and development is still going on. Support in this stage is provided by the Engineering team, and is only available to members of the program.
+
+## Early Acess
+
+Early Access projects are similar to Pilot projects, but are a little more mature, and have been opened up for use by a larger number of users. An invitation is still necessary to join an Early Access program.
```
Comment (kporras07) [inline source/content/oss-support-levels.md:16]: Thank you! I had a hard time with this one 😂

**Comment by greg-1-anderson** [comment]
LGTM, ready for docs review

File: source/content/oss-support-levels.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+A project that has unofficial support level is usually a Pantheon employee initiative. It typically does not have resources allocated for maintenance. Support is not guaranteed for the project because it depends on the availability and afffiliation of the project maintainer.
+
+## Unsupported 
+
+An unsupported project is typically only for internal use at Pantheon. Although the source code is available, use these projects at your own risk, as support is not available.
+
+## Deprecated 
+
+A deprecated project is a project that has been retired or replaced by a code base that is no longer supported by Pantheon. Details about the project can be found in the `README.md` or you could try asking us in the [Community Slack](https://slackin.pantheon.io/) if you are in a need for the replacement or guidance about where to go. Using a project that has been deprecated is not encouraged.
```
Comment (greg-1-anderson) [inline source/content/oss-support-levels.md:38]: "replaced by a code base that is no longer supported by Pantheon" no longer modifies the correct object. The deprecated project is no longer supported by Pantheon; the code base that replaces it probably is.

Maybe: "replaced by some other project, and is no longer supported by Pantheon."

**Review by joan-ing — APPROVED** [review]
LGTM

**Pull request merged**